### PR TITLE
Add a we've moved banner to doc site

### PR DESCRIPTION
**Pull request by MaxWang-MS**
## Overview
Add a we've moved banner to all conceptual doc and API reference pages on GitHub.io doc site.
image

### CODE REVIEW

File: scripts/docs/templates/mrtk/partials/navbar.tmpl.partial [hunk 1]
```diff
@@ -23,4 +23,12 @@
...
     </div>
   </div>
-</nav>
\ No newline at end of file
+</nav>
+  <div class="navbar alert-navbar">
+  <div class="container">
+      <div class="alert-navbar-message">
+        <h5>We've moved!</h5>
+        <p>Starting from MRTK 2.6, we are publishing both conceptual docs and API references on docs.microsoft.com. For conceptual docs, please visit <a href="https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/">our new landing page</a>. For API references, please visit <a href="https://docs.microsoft.com/dotnet/api/microsoft.mixedreality.toolkit">the MRTK-Unity section of the dot net API explorer</a>. Existing content here will remain but no further update is expected.</p>
```
Comment (RogPodge) [inline scripts/docs/templates/mrtk/partials/navbar.tmpl.partial:31]:
```suggestion
        <p>Starting from MRTK 2.6, we are publishing both conceptual docs and API references on docs.microsoft.com. For conceptual docs, please visit <a href="https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/">our new landing page</a>. For API references, please visit <a href="https://docs.microsoft.com/dotnet/api/microsoft.mixedreality.toolkit">the MRTK-Unity section of the dot net API explorer</a>. Existing content will remain here but no further update is expected.</p>
```

**Review by RogPodge — CHANGES_REQUESTED** [review]

**Review by RogPodge — APPROVED** [review]

**Comment by polar-kev** [comment]
Looks good to me.

**Review by polar-kev — APPROVED** [review]

**Comment by keveleigh** [comment]
Do we want to cherry-pick this into 2.5.4, our current default docs landing page for github.io?

**Comment by MaxWang-MS** [comment]
Yes and I am actually thinking whether we should cherry pick this into all previous releases. @keveleigh @polar-kev

**Comment by polar-kev** [comment]
Yes, please do. @MaxWang-MS @keveleigh

**Pull request merged**